### ISSUE TITLE
Can't load the libraries

### ISSUE THREAD

**Issue by mspr1nt** [Joexv/HeadRipper#4]
Hello. Firstly, thanks for this, it is MASSIVELY helpful. I love Headspace but get super anxious on some nights and even more so knowing a Sleepcast might finish before I fall asleep.

That said, I can't seem to get this to work anymore. I'm getting a strange error, part of it reads as follows::  Could not load file or assembly 'Newtonsoft.Json, Version=12.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed' or one of its dependencies. The system cannot find the file specified

I have a feeling I'm putting things in a wrong folder somewhere, so any help appreciated please.

**Comment by Joexv** [comment]
Make sure when you extract the program you extract all the DLL files along with it. That error means it’s missing them.

**Comment by mspr1nt** [comment]
Thanks - but I'm not even 100% sure how exactly to install/extract it - I think I very much fluked it when it was working before. 
Because just unzipping the file doesn't work even if everything is kept in the same folder.

**Comment by Joexv** [comment]
Interesting okay. And you have the BearerID in the program?

Could you re-zip what you have and send it over?

**Comment by mspr1nt** [comment]
Indeed. I also fetched the bearerID again just to check if I maybe got a rogue copy somewhere (like I said, it has worked before but how I managed it I have no idea). I also added it and ffmpeg to exclusions on Kaspersky in case there was interference. One of the manoeuvres ended with the simple error of "unable to load categories" (which is one step further than before, but a similar error returned a short while later).  Anyways, I'm not super familiar with Github and for the life of me cannot figure out how to attach a rezip.
